config/generic-mellow-fly-d5.cfg

```diff
@@ -0,0 +1,158 @@
+# This file contains common pin mappings for the Mellow Fly D5
+
+# Important! Do not use this config with a Fly DP5 board!
+
+# To use this board in serial mode, during "make menuconfig", select "Enable
+# low-level configuration options", select the STM32F072
+# micro-controller, select a "8KiB bootloader", and select a "8Mhz
+# crystal", "USB (On PA11/PA12)"
+
+# To use this board in CAN bus bridge mode(along with a CAN bus toolhead
+# board), during "make menuconfig", select "Enable low-level
+# configuration options", select the STM32F072 micro-controller,
+# select a "8KiB bootloader", and select a "8Mhz crystal",
+# "USB to CAN bus bridge (USB on PA11/PA12)". In this mode, the mainboard
+# it self will also work in CAN bus mode, so you will need a canbus_uuid
+# instead of serial.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[mcu]
+serial: /dev/serial/by-id/usb-klipper-stm32f072xx-1234567890987654321 # Replace with your serial address here, see docs/FAQ.md#wheres-my-serial-port for more
+#canbus_uuid: 11aa22bb33cc # Replace with your canbus uuid here, see docs/CANBUS.md for more
+
+[temperature_sensor Fly-D5] # Mainboard Temperature
+sensor_type: temperature_mcu
+
+[printer]
+kinematics: corexy # Replace with your machine's kinematic!(e.g. cartesian/delta)
+max_velocity: 350
+max_accel: 10000
+max_z_velocity: 10
+max_z_accel: 200
+
+[extruder]
+step_pin: PC5
+dir_pin: !PC4
+enable_pin: !PB0
+microsteps: 64
+nozzle_diameter: 0.4
+filament_diameter: 1.75
+rotation_distance: 23.310912
+gear_ratio: 50:10
+heater_pin: PC6
+sensor_pin: PC1
+sensor_type: Generic 3950
+control: pid # Run "PID_CALIBRATE HEATER=extruder TARGET=250" to calibrate flowing pid control values before print! They are just placeholders!
+pid_Kp: 10
+pid_Ki: 10
+pid_Kd: 10
+min_temp: 0
+max_temp: 300
+
+#[tmc2209 extruder] # Uncomment this section if using a TMC2209 stepper driver, replace it with your TMC model. Applies to the other [tmc2209] section.
+#uart_pin: PA7
+#interpolate: True
+#run_current: 0.8
+#stealthchop_threshold: 0
+
+[stepper_x]
+step_pin: PC15
+dir_pin: PC14
+enable_pin: !PC2
+microsteps: 64
+rotation_distance: 40   # See docs/Rotation_Distance.md
+endstop_pin:^PB4
+position_endstop: 0
+position_max: 200
+
+#[tmc2209 stepper_x]
+#uart_pin: PC13
+#cs_pin: PC13 # References for SPI TMC drivers(tmc5160, tmc2240, etc.)
```

Comment:
While I agree that this is an efficient way, my concern is that a regular user will only be confused.

The clearest way IMO is to keep the TMC settings together in a way it is done here: https://github.com/Klipper3d/klipper/blob/master/config/generic-bigtreetech-manta-m8p-v1.1.cfg